openhands_cli/refactor/widgets/richlog_visualizer.py

```diff
@@ -361,6 +362,11 @@ def _create_event_collapsible(self, event: Event) -> NonClickableCollapsible | N
 
     def _format_metrics_subtitle(self) -> str | None:
         """Format LLM metrics as a visually appealing subtitle string."""
+        # Check app configuration to see if metrics should be displayed
+        app_config = AppConfiguration.load()
+        if not app_config.display_cost_per_action:
+            return None
+
```

Comment:
@openhands note that this seems very inefficient, its reading from the filestore every single time

please make sure that we're caching or doing a more efficient update of the configuration such that we aren't constantly loading the configurations